packages/flutter/test/material/tabs_test.dart

```diff
@@ -2035,7 +2035,7 @@ void main() {
     await tester.pumpAndSettle();
     await tester.pump();
     expect(controller.index, 3);
-    expect(find.text(AlwaysKeepAliveWidget.text, skipOffstage: false), findsOneWidget);
+    expect(find.text(AlwaysKeepAliveWidget.text), findsOneWidget);
```

Comment:
Why this change? (i'd expect the test to fail now)

Reply:
I was playing around the test, It is weird that this test should have failed at the first place, otherwise it would catch the bug